templates/observation.html

```diff
@@ -851,6 +846,63 @@ <h5 class="modal-title" id="confirmModalLabel">Reset media</h5>
         selection: false
       });
 
+      const canvasHeight = 332;
+      const canvasWidth = 530;
+
+      if (!is_touch_device) {
+        stage.on('mouse:wheel', function(opt) {
+          var delta = opt.e.deltaY * -1;
+          var pointer = stage.getPointer(opt.e);
+          var zoom = stage.getZoom();
+          zoom = zoom + delta / 200;
+
+          if (zoom > 10) {
+            zoom = 10;
+          }
+
+          if (zoom < 1) {
+            zoom = 1;
+          }
+
+          stage.zoomToPoint({ x: opt.e.offsetX, y: opt.e.offsetY }, zoom);
+          opt.e.preventDefault();
+          opt.e.stopPropagation();
+
+          const vpt = this.viewportTransform;
+
+          if (zoom < 200 / canvasWidth) {
```

Comment:
What are this magic number: `200`, `120`, `[4]`, `[5]`. IMHO it would be cool to add aliases for them.